src/mono/wasm/debugger/BrowserDebugProxy/ValueTypeClass.cs

```diff
@@ -63,34 +67,45 @@ public ValueTypeClass(byte[] buffer, string className, JArray fields, int typeId
             IReadOnlyList<FieldTypeClass> fieldTypes = await sdbAgent.GetTypeFields(typeId, token);
 
             JArray fields = new();
+            List<JObject> inlineArray = null;
+            JObject fieldValue = null;
             if (includeStatic)
             {
                 IEnumerable<FieldTypeClass> staticFields =
                     fieldTypes.Where(f => f.Attributes.HasFlag(FieldAttributes.Static));
                 foreach (var field in staticFields)
                 {
-                    var fieldValue = await sdbAgent.GetFieldValue(typeId, field.Id, token);
+                    fieldValue = await sdbAgent.GetFieldValue(typeId, field.Id, token);
```

Comment:
nit: Keep this local name, so there's no chance of this interfering with the `writableFields`.